frontend/src/Components/Left_navbar_components/Accordion.svelte

```diff
@@ -1,8 +1,8 @@
 <script lang="ts">
   export let title: string;
-  export let color: string = "#0d5474";
+  export let color: string = "#004372";
```

Comment:
Maybe we could use one of the colors defined in tailwindcss?

Reply:
Can have a look at that later 8)